health/micro-ui/web/health-payments/Dockerfile

```
@@ -5,20 +1,42 @@
+FROM node:14-alpine3.14 AS build
+RUN apk update
+RUN apk add --no-cache 'git>2.30.0'
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)